File: bubble.py
def bubble (arr):
    """
    Bubble function sort an list with bubble sort method.
    Bubble sort method bring the biggest value of element to the end of the list
    """
    n = len(arr)
    for i in range(n):
        swapped = False
        # end of list is already sorted
        for j in range(n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
                swapped = True
        # rest of the array is sorted
        if swapped is False:
            break
    return arr



print(bubble([4, 20, 12, 10, 7, 9]))

File: bubble.js
function bubbleSort(arr) {
    let swapped, temp
    for(let i=0; i<arr.length; i++) {
        swapped = false
        for(let j=0; j<arr.length-i-1; j++){
            if (arr[j] > arr[j+1]) {
                temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
                swapped = true
            }
        }
        if (swapped == false) break
    }
    console.log(arr)
    return arr
}

module.exports = bubbleSort;